easybuild/easyconfigs/s/shapeit/shapeit-2.r837.GLIBCv2.12_linux_x86_64.eb

```diff
@@ -0,0 +1,24 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'shapeit'
+version = '2.r837.GLIBCv2.12'
+versionsuffix = '_linux_x86_64'
+
+homepage = 'https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html'
+description = """ SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data.  """
```

Comment:
@rtripath89 we strive to limits line length to max 120 characters, so please split this up (and drop the extra spaces while you're at it):

```python
+description = """SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing)
 from genotype or sequencing data."""
```